conf/deployment/aws/ipi_1az_rhcos_lso_3m_3w.yaml

```diff
@@ -14,3 +14,6 @@ ENV_DATA:
   master_replicas: 3
   worker_instance_type: 'i3en.2xlarge'
   device_size: 2300  # Value pulled from here: https://bugzilla.redhat.com/show_bug.cgi?id=1812557#c27
+REPORTING:
+  polarion:
+    deployment_id: 'OCS-2196'
```

Comment:
I haven't seen this format, what functionality adds the polar ion-id here?

Reply:
We have this in our other production configurations. See this example: https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/aws/ipi_1az_rhcos_3m_3w.yaml#L15-L17

The polarion ID is read in [here](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/reporting.py#L20) and added as a property on the test case for our report.xml [here](https://github.com/red-hat-storage/ocs-ci/blob/36c47d6bd2772fb2bc612196da07409b6b4a3436/tests/ecosystem/deployment/test_deployment.py#L14)